packages/relayer/docker-compose/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
     restart: always
     environment:
       - MYSQL_DATABASE=relayer
-      - MYSQL_ROOT_PASSWORD=<password>
+      - MYSQL_ROOT_PASSWORD=passw00d
```

Comment:
not supposed to use even default passwords according to Daniel

Reply:
Reverted.